test/CMakeLists.txt

```diff
@@ -22,12 +22,22 @@ configure_lit_site_cfg(
 set(NVQPP_TEST_PARAMS
   nvqpp_site_config=${CMAKE_CURRENT_BINARY_DIR}/lit.site.cfg.py)
 
+# Gather all the runtime libraries to make sure they are up-to-date.
+# FIXME: These libraries should self-register to a CMake property so here we
+# can just use a macro instead of listing them.
+set(TEST_CUDAQ_LIBRARIES
+  cudaq cudaq-common cudaq-mlir-runtime cudaq-builder cudaq-ensmallen
+  cudaq-nlopt cudaq-spin
+  cudaq-em-default cudaq-em-photonics cudaq-platform-default
```

Comment:
cudaq-em-default and cudaq-em-photonics should not be used together - they both provide an ExecutionManager.

Reply:
I can drop the photonics, but these are meant as build dependencies. Do we know if the regression tests depend on both libraries being built or not?